django_test_migrations/checks/autonames.py

```diff
@@ -78,7 +78,10 @@ def check_migration_names(*args, **kwargs) -> List[CheckMessage]:
                         "or if it's from a third party app, add to " +
                         _SETTINGS_NAME
                     ),
-                    id='{0}.E001'.format(CHECK_NAME),
+                    id='{0}.W001'.format(CHECK_NAME),
```

Comment:
Why do we change the check name? (I don't remember the context behind it)

Reply:
Mainly because it's a `Warning` not `Error` and in django there is a naming convention, that all `Info`  messages id suffixes starts with `I`, all `Warning` with `W` etc

Reply:
Oh, I see! Thanks a lot for the explanation.